assets/js/modules/analytics-4/datastore/webdatastreams.js

```diff
@@ -144,6 +174,18 @@ const baseResolvers = {
 			yield fetchGetWebDataStreamsStore.actions.fetchGetWebDataStreams( propertyID );
 		}
 	},
+	*getWebDataStreamsBatch( propertyIDs ) {
+		const registry = yield Data.commonActions.getRegistry();
+		const webdatastreams = registry.select( STORE_NAME ).getWebDataStreamsBatch( propertyIDs ) || {};
+
+		const availablePropertyIDs = Object.keys( webdatastreams );
+		const remainingPropertyIDs = difference( propertyIDs, availablePropertyIDs );
+		if ( remainingPropertyIDs.length > 0 ) {
+			for ( let i = 0; i < remainingPropertyIDs.length; i += 10 ) {
```

Comment:
Since the per-request limit only exists on the server so far, how about we extract the `10` to a constant such as `MAX_WEBDATASTREAMS_PER_BATCH` ? That would make this a bit clearer why we're using this value specifically.

Reply:
Updated.